src/server/document-controller.js

```diff
@@ -105,6 +106,10 @@ class DocumentController {
       room = null;
     }
 
+    if (view === DOC_VIEW_QUERY_PARAM.edit && !canEditDocContent({ user, doc, room })) {
+      return res.redirect(routes.getDocUrl({ id: doc._id, slug: doc.slug, templateDocumentId }));
```

Comment:
As the `view` param is not (and cannot be) "edit" here, we should also ditch any possible `templateDocumentId` from the URL here, I guess.

Reply:
I wanted to keep everything but the `view` param, but sure, I can also simplify.

Reply:
Done!

Reply:
Cool, thanks!